--- ./node_modules/jest-config/build/normalize.js.org	2019-03-24 09:27:15.000000000 +0200
+++ ./node_modules/jest-config/build/normalize.js	2019-03-24 09:47:07.000000000 +0200
@@ -199,7 +199,8 @@
   );
 
   try {
-    // Force re-evaluation to support multiple projects
+    // Force re-evaluation to support multiple project
+    console.log('Setup presetModule', presetModule);
     try {
       if (presetModule) {
         delete require.cache[require.resolve(presetModule)];
@@ -208,6 +209,7 @@
 
     preset = require(presetModule);
   } catch (error) {
+    console.log('Error setting up preset', error);
     if (error instanceof SyntaxError || error instanceof TypeError) {
       throw createConfigError(
         `  Preset ${_chalk().default.bold(presetPath)} is invalid:\n\n  ${
